src/main/java/hudson/plugins/sidebar_link/ProjectLinks.java

```diff
@@ -24,14 +24,14 @@
 package hudson.plugins.sidebar_link;
 
 import hudson.Extension;
-import hudson.model.AbstractProject;
-import hudson.model.Action;
-import hudson.model.JobProperty;
-import hudson.model.JobPropertyDescriptor;
+import hudson.model.*;
```

Comment:
Please do not do that in the production code. At some point it may corrupt the code with false positive. E.g. if one writes a hudson.model.List class and ignores the ambiguity warning. It is not often, but better safe than sorry...